src/main/java/seedu/taassist/model/TaAssist.java

```diff
@@ -175,6 +181,51 @@ public void removeSession(ModuleClass moduleClass, Session session) {
         setModuleClass(moduleClass, moduleClass.removeSession(session));
     }
 
+    /**
+     * Exports the {@code moduleClass} from this {@code TaAssist} as a CSV file.
+     * {@code moduleClass} must exist in TA-Assist.
+     * @return The CSV file object created.
```

Comment:
Perhaps should not say "the CSV file object created" since no such object is explicitly created. Maybe something along the lines of "The 2D stringified list of student session data for the specified class". Perhaps can also clarify that doesn't actually return a CSV file object, but rather this method is used to parse data used for such purposes.

Also maybe for abstraction reasons, should Model really be handling how objects are "displayed"? Should we just return a list of Students for this specific class and let Logic handle the actual data parsing? (This should be possible with methods such as `getStudentModuleData`.

Reply:
This issue is a broader issue and can be ignored for now. We can consider improving some aspects of our abstraction in v1.4

Reply:
Done in next commit.